WcaOnRails/app/models/competition.rb

```diff
@@ -314,6 +328,10 @@ def end_date
     endYear == 0 || endMonth == 0 || endDay == 0 ? nil : Date.new(endYear, endMonth, endDay)
   end
 
+  def marker_date
+    start_date.strftime("%b %d, %Y")
```

Comment:
Date formatting rules shouldn't be here. I want to make sure things stay consistent across the site. Can we use `to_formatted_s` here instead? See https://github.com/cubing/worldcubeassociation.org/blob/37ea8a6b71c6a9cd855674efc12afce0589ea154/WcaOnRails/app/views/registrations_mailer/notify_registrant_of_accepted_registration.html.erb#L9.